docscan/session/create/check/requested_check.go

```diff
@@ -1,11 +1,12 @@
 package check
 
-// RequestedCheck requests creation of a Check to be performed on a document
-type RequestedCheck struct {
-	Type   string               `json:"type"`
-	Config RequestedCheckConfig `json:"config"`
+// RequestedChecker requests creation of a Check to be performed on a document
+type RequestedChecker interface {
```

Comment:
I don't think this needs the `-er` suffix as it's not a one-method interface

Reply:
I would actually take this one step further and say it doesn't add value, since it's empty. It would be fine to just use `interface{}`

Reply:
Good catch, reverted in 0e9669d